### PR TITLE
Adding SNS IntegrationSink example

### PR DESCRIPTION
**Pull request by matzew**
## Proposed Changes 

- adding docs for the `sns` layer of the `IntegrationSink` CRD

### CODE REVIEW

File: docs/eventing/sinks/integration-sink/aws_sns.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# AWS Simple Notification Service Sink
+
+The `IntegrationSink` supports the Amazon Web Services (AWS) Simple Notification Service (SNS) service, through its `aws.sns` property.
+
+## Amazon credentials
+
+For connecting to AWS the `IntegrationSink` uses Kubernetes `Secret`, present in the namespace of the reSink. The `Secret` can be created like:
```
Comment (pierDipi) [inline docs/eventing/sinks/integration-sink/aws_sns.md:7]:
```suggestion
For connecting to AWS the `IntegrationSink` uses Kubernetes `Secret`, present in the namespace of the sink resource. The `Secret` can be created like:
```

Comment (pierDipi) [inline docs/eventing/sinks/integration-sink/aws_sns.md:7]: will we document the permissions to give to those access keys?

Comment (matzew) [reply docs/eventing/sinks/integration-sink/aws_sns.md:7]: That's perhaps a good idea. I will see how camel handles this, on their docs, and borrow.

**Comment by matzew** [comment]
@pierDipi updated. same typo was in the other sinks - fixed them too in here

**Review by pierDipi — APPROVED** [review]
/lgtm

**Review by dsimansk — APPROVED** [review]
/approve
/lgtm

**Pull request merged**